opcua/client.py

```diff
@@ -307,3 +307,6 @@ def get_namespace_array(self):
     def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
+
+    def history_read(self, params):
+        return self.bclient.history_read(params)
```

Comment:
the Client class is supposed to be a very high level class and expose BinaryClient for users who want more control, so there is no need to expose the same method in Client and BinaryClient. (btw maybe we should rename bclient to binary_client to make it more discoverable)
I do not remember well the history API but I think that the history_read or better read_history method should be moved to Node class. Maybe just move read_raw_history to Node and call it read_history, if you think it makes sense